docs/user_guide.md

```diff
@@ -402,3 +402,40 @@ you can see the item named "flink-pod-monitor" in the "Service Discovery" sectio
 ### Manage savepoints
 
 See this [doc](./savepoints_guide.md) on how to manage savepoints with the operator.
+
+### Update Flink clusters and jobs
+
+To update a running Flink job's program or execution settings, you can create new savepoint, terminate the job,
+and create a new FlinkCluster with the new program, settings, and savepoint. 
+However, in some cases, you may want to update the Flink job while maintaining the logical continuity
+of the Flink job with the FlinkCluster custom resource. In this case, you can continuously update
+the FlinkCluster custom resource, and the Flink operator takes care of the process required to update
+the Flink job and cluster.
+
+There are several points to note when using the job update feature.
+* To use the job update feature, `savepointsDir` must be set and the value of this field cannot be deleted when updating.
+This is because the Flink operator requires it to create a savepoint for job updates.
+* You can resume Flink job from your desired savepoint by updating `fromSavepoint`.
+If you want to resume the updated job from the latest savepoint, `fromSavepoint` must be unspecified.
+* `cancelRequested` and `savepointGeneration` are not allowed to update at the same time with other fields
+due to functional characteristics.
+
+There are some behavioral characteristics in update.
+* Whenever the FlinkCluster spec is updated, the Flink operator creates
+a [ControllerRevision](https://godoc.org/k8s.io/api/apps/v1#ControllerRevision) resource
+that stores the changed spec. ControllerRevisions can be used to check the editing history.
```

Comment:
Add command-line examples on how to check the history.

Reply:
Done.